test/remote/gateways/remote_cecabank_test.rb

```diff
@@ -36,20 +36,19 @@ def test_successful_refund
   end
 
   def test_unsuccessful_refund
-    assert response = @gateway.refund(@amount, 'wrongreference', @options)
+    purchase = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success purchase
+
+    assert response = @gateway.refund(@amount, purchase.authorization, @options.merge(currency: 'USD'))
     assert_failure response
-    assert_equal 'ERROR', response.message
+    assert_match 'Error de comunicaciones', response.message
   end
 
   def test_invalid_login
-    gateway = CecabankGateway.new(
-      :merchant_id => '',
-      :acquirer_bin => '',
-      :terminal_id => '',
-      :key => ''
-    )
+    gateway = CecabankGateway.new(fixtures(:cecabank).merge(key: 'invalid'))
```

Comment:
Here, I updated the `gateway` definition to supply only an invalid `key` because when all of the other parameters were invalid, the response was ""Unable to parse the response. #<ArgumentError: invalid byte sequence in UTF-8>" along with a lot of other HTML. 

This way, the error message is "ERROR AL CALCULAR FIRMA." ("Error in calculating the signature."), which seems much more helpful and appropriate.